src/utils/api.js

```diff
@@ -29,6 +29,34 @@ export const formatActions = (actionsByDayObject) => {
   });
 };
 
+// 'usersArray' has the form [ { ids: [1, 2], name: 'Augie March', type: 'light', value: 'Augie March'}, {...}, ... ]
+// i.e. a user (identified by their name) can have multiple ids (due to different sign-in sesions)
+// 'actions' is an array in the format retrieved from the API: [ { id: 1, user: 2, ... }, {...} ]
+// therefore note: id is the id of the action, and user is the userId of the user performing the action
+export const filterActionsByUser = (actions, usersArray) => {
+  return actions.filter((action) => {
+    return usersArray.some((user) => {
+      return user.ids.includes(action.user);
+    });
+  });
+};
+
+// given an actionsByDay object, findYAxisMax finds max value to set on the yAxis in the graph in ActionsChart.js
+export const findYAxisMax = (actionsByDay) => {
+  const arrayOfActionsCount = Object.values(actionsByDay);
+  if (!arrayOfActionsCount.length) {
+    return null;
+  }
+  const maxActionsCount = arrayOfActionsCount.reduce((a, b) => Math.max(a, b));
+  let yAxisMax;
```

Comment:
Why do you have these two cases?

Reply:
if maxActionsCount <= 100 (e.g. we select a user who has 6 actions), we round up yAxisMax to the nearest ten, but if > 100, to the nearest hundred. I added a comment to clarify.